arch/arm/src/stm32/stm32_sdio.c

```diff
@@ -172,9 +172,14 @@
 #define SDIO_CMDTIMEOUT          (100000)
 #define SDIO_LONGTIMEOUT         (0x7fffffff)
 
-/* Big DTIMER setting */
+/* DTIMER setting
+ *
+ * Assuming Max timeout in bypass 48 Mhz
+ *
+ */
```

Comment:
```suggestion
/* DTIMER setting
 *
 * Assuming Max timeout in bypass 48 Mhz
 */
```
This empty line may be removed.

Reply:
Reformatted